lib/smart_app_launch/cors_metadata_request_test.rb

```diff
@@ -8,7 +8,8 @@ class CORSMetadataRequest < Inferno::Test
 
     title 'SMART metadata Endpoint Enables Cross-Origin Resource Sharing (CORS)'
     description %(
-      For requests from any origin, CORS configuration permits access to the public discovery endpoints
+      The SMART [Considerations for Cross-Origin Resource Sharing (CORS) support](http://hl7.org/fhir/smart-app-launch/STU2.2/app-launch.html#considerations-for-cross-origin-resource-sharing-cors-support)
```

Comment:
These descriptions are all missing the qualifier that these requirements are only for servers which support purely browser-based apps.